### ISSUE TITLE
In training mode, error: invalid choice.

### ISSUE THREAD

**Issue by wjianga** [bioinform/somaticseq#95]
Hi,

I have the following error message when I tried to run the training mode. While I imitated the syntax in the manual and specified the "single" mode, I cannot understand the error message, especially the fact that it is pointing to my "ground truth" vcf files.

Error message:
somaticseq_parallel.py: error: invalid choice: 'ALMC1.SNPs_only.recode.vcf' (choose from 'paired', 'single')

Commands:
somaticseq_parallel.py \
--output-directory ALMC1_DJ_p27_CL_Whole_T1_A2S4U_K03485_D1KLRACXX_AACCGAGA \
--genome-reference GRCh38.p7.genome.fa \
--truth−snv ALMC1.SNPs_only.recode.vcf \
--truth−indel ALMC1.Indels_only.recode.vcf \
--somaticseq−algorithm xgboost \
--somaticseq−train \
single \
--bam-file ALMC1_DJ_p27_CL_Whole_T1_A2S4U_K03485_D1KLRACXX_AACCGAGA.recalibrated.bam \
--varscan-vcf ALMC1_DJ_p27_CL_Whole_T1_A2S4U_K03485_D1KLRACXX_AACCGAGA.vcf \
--mutect2-vcf ALMC1_DJ_p27_CL_Whole_T1_A2S4U_K03485_D1KLRACXX_AACCGAGA.filterdvar.vcf

Thanks!

**Comment by litaifang** [comment]
Try replacing `--somaticseq-algorithm xgboost` with `--algorithm xgboost`.

**Comment by wjianga** [comment]
It is still reporting the same error.

**Comment by litaifang** [comment]
Try typing out the command again. Make sure the hyphens are real hyphens `-`, instead of some characters that look like hyphens. 
I copy-and-pasted the commands and it didn't work, and noticed there is some difference in the hyphens between I type and ones I copied, i.e, `--truth−snv` vs. `--truth-snv`.

**Comment by wjianga** [comment]
It works now. The error was due to a wrong hyphen. Thank you!
